packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -218,11 +218,13 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
       groups
     });
 
+    const finalExpandBtnAriaLabel = expandButtonAriaLabel ? `${group.name} ${expandButtonAriaLabel}` : group.name;
```

Comment:
This could be a localization concern. Is the subject before the verb in all languages?

Reply:
Good call. This is the way other groups were being handled, but I think you're right that this is not ideal. Introduced some new attributes on links/groups to handle this correctly and deprecated the old method.